utils/scripts/install-el8.sh

```diff
@@ -62,7 +62,6 @@ dnf --nodocs install ${dnf_install_args} \
     ndctl-devel \
     numactl \
     numactl-devel \
-    openmpi-devel \
```

Comment:
Are we sure we don't need this?

Reply:
I'm just trying it to see if it passes the test. I'm not sure if this is an effective fix or not. The openmpi that installs with DOCA-OFED does not allow installing of an openmpi-devel rpm. The openmpi-devel rpm installs the module file for doing module load. This is needed for CI execution of daos_test/suite.yml to run. @JohnMalmberg change the install_mellanox.sh to create the openmpi module file as part of the DOCA-OFED install.

Reply:
If everything is working without this then it might be okay to leave it out permanently. It might be that we needed it at one point but do not anymore.

Reply:
The install-el8.sh script has been changed install the openmpi-devel rpm by default, but the change enables an option to skip this installation. The daos_build.py code has been changed to set this option when invoking the install script.

Reply:
The -devel files are only used at build time, not a run time.

If a test is doing some type of build it requires a devel file.

Mellanox is just building the RPMs from the spec files in the upstream OpenMPI project that do not match either the RedHat or SUSE packaging of OpenMPI.   The missing "-devel" packages is not the only critical thing they are missing, as we have to create the module file that those RPMs should be supplying.

If someone could get a PR landed to the upstream OpenMPI project to fix the spec files, it would fix a bunch of compatibility problems with doing builds after installing Mellanox software.

Has someone filed a ticket with Mellanox about them building incompatible OpenMPI rpms?